x-pack/plugins/index_lifecycle_management/public/extend_index_management/components/index_lifecycle_summary.tsx

```diff
@@ -243,3 +246,18 @@ export const IndexLifecycleSummary: FunctionComponent<Props> = ({ index, getUrlF
     </>
   );
 };
+
+export const indexLifecycleTab: IndexDetailsTab = {
+  id: 'ilm',
+  name: (
+    <FormattedMessage
+      defaultMessage="Index Lifecycle"
```

Comment:
I _think_ this should be sentence case 

```suggestion
      defaultMessage="Index lifecycle"
```